bin/broker-server-start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Any value in trying to reuse/refactor the existing script to keep this in sync with kafka-server-start without much additional effort? Not sure if it would make things better or worse than it would be with this approach since it might just mean painful merges each time anything changes upstream.